build-logic/idea/src/main/kotlin/gradlebuild.ide.gradle.kts

```diff
@@ -81,12 +81,21 @@ fun ProjectSettings.configureCopyright() {
 
 fun ProjectSettings.configureRunConfigurations() {
     runConfigurations {
-        create<Remote>("Remote debug port 5005") {
-            mode = Remote.RemoteMode.ATTACH
+        create<Remote>("Daemon debug (port 5005)") {
```

Comment:
🤔 Doesn't IDEA try to attach a debugger to the test jvm that is different from the Daemon JVM on port 5005?

Reply:
IntelliJ doesn't know if it's a daemon or some other JVM, sure. But I think it's easier for humans to select as a default option, even if it can be reused in other situations

Reply:
What I mean is that you want to debug your test as well from IDEA. And IDEA will use port 5005 for that when you press the debug button, right? Or does IDEA use an arbitrary port?

Reply:
It works fine with test debugging. The test debug/breakpoints are shown in a "regular" configuration (the one you use to run the test), while deamon debug/breakpoints are shown in "Daemon Debug". So they don't interfere with each other. FTR, you can debug the test code, the daemon and the launcher at the same time.